numpy/add_newdocs.py

```diff
@@ -547,10 +547,26 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
-add_newdoc('numpy.core', 'broadcast', ('nd',
+add_newdoc('numpy.core', 'broadcast', ('ndim',
     """
     Number of dimensions of broadcasted result.
 
+    .. versionadded:: 1.12.0
+
```

Comment:
Should have a `See Also` section to reference `nd`, search for examples, and `nd` should also get the section to reference `ndim`. In the summary explain that `ndim` is an alias of `nd` intended to improve commonality with `ndarray` and `nditer`.